ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.15.yaml

```diff
@@ -149,6 +149,15 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-xpn-cco-manual-users-static
+- as: nutanix-ipi-realtime-f28
```

Comment:
I think we also can add this case to openshift-verification-tests-master__installation-nightly-4.14.yaml